app/views/publish/index.html.erb

```diff
@@ -18,14 +18,14 @@
       <%# span class="time version"><%= l(service.created_at.to_time, format: :time) %></span %>
       <%# span class="name by"><%= t('publish.test.by', name: 'C. Smith') %></span %>
     <%# /p %>
+    <p><%= PublishServicePresenter.hostname_for(deployment_environment: 'dev', view: self) %></p>
 
-    <span>
-      <%= PublishServicePresenter.hostname_for(deployment_environment: 'dev', view: self) %>
-    </span>
-
+    <%# TODO: Need an indicator from BE code that shows whether this is first-time publish   %>
+    <%#       Currently have hardcoded 'false' value to avoid dialog showing but this should %>
+    <%#       be dynamic to control dialog visibility.                                        %>
     <%= form_for(@publish_service_creation_dev,
         url: publish_index_path(service.service_id),
-        html: { id: 'publish-form-dev', class: 'publish-form' }) do |f| %>
```

Comment:
@tomas-stefano 
Notice how I've hardcoded a value here.
`data: { firstpublish: false }`

If we (and by that I mean you - hehe! ) can make this dynamic then we will have a completed controlling mechanism for showing the '15 minute' dialog message on initial publish (make it true, publish a form, and you will see it).

Let me know if that's clear as mud.

Reply:
surely you mean `first_publish`? 😜

Reply:
Cool, agree with above on `first_publish`. Happy to merge this PR and add the dynamic in a separate one. :)